recipe/bld.bat

```diff
@@ -0,0 +1,6 @@
+set TCL_LIB_PATH=%PREFIX%\Library\lib\tcl8.6\%PKG_NAME%%PKG_VERSION%
```

Comment:
hardcoded versions in scripts are always risky. it is probably better to export a visible env var from the recipe. or somehow recover the current lib path from tcl/tk itself.